grails-app/services/io/xh/toolbox/security/Auth0Service.groovy

```diff
@@ -17,17 +20,13 @@ import static io.xh.hoist.util.InstanceConfigUtils.getInstanceConfig
 @Slf4j
 class Auth0Service extends BaseService {
 
+    private JSONClient _jsonClient
 
-    static clearCachesConfigs = ['auth0Domain', 'auth0ClientId', 'auth0Jwks']
+    static clearCachesConfigs = ['auth0Domain', 'auth0ClientId']
 
     ConfigService configService
 
     Map getClientConfig() {
```

Comment:
it occurs to me that, we could actually even just go back to having the server disable this if the bootstrap user is set, and then not worry about any of the other logic!  Or even better do both -- e.g. throw here if no bootstrapUser set and  domains are not in the whitelist that will allow oAuth to work....